### ISSUE TITLE
`pick` error 

### ISSUE THREAD

**Issue by ixzh** [sharkdp/pastel#153]
Hi,
I'm having this error when running`pastel pick`:
> Error while running color picker 'gdbus
I install pipette via pip, not helping. how can I get gdbus to work?

**Comment by sharkdp** [comment]
@sigmaSd Thank you very much for the prompt response. Lowering the precedence will help. And is something we should probably do. I didn't realize it either during the review.

However, I don't think that's enough. The problem is that `gdbus` exists on our systems, but the call fails. This is not something that happens with the other color pickers. If they exist, and if we can call them successfully with the version argument and get a proper response, we expect them to succeed.

It seems like for `gdbus`, we need a more sophisticated workflow. On our machines, it fails with:
```
Error: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.Shell.Screenshot was not provided by any .service files
```
Maybe there is some other call we could run first to see if the call will succeed? Or we simply run a similar command as "version command" and skip `gdbus` as a picker if that call fails?

**Comment by sharkdp** [comment]
Closed via #155.
